frontend/packages/kubevirt-plugin/src/topology/components/nodes/VmNode.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Tooltip } from '@patternfly/react-core';
+import { VirtualMachineIcon } from '@patternfly/react-icons';
+import {
+  Node,
+  observer,
+  WithSelectionProps,
+  WithContextMenuProps,
+  WithCreateConnectorProps,
+  WithDragNodeProps,
+  WithDndDropProps,
+  useCombineRefs,
+  createSvgIdUrl,
+  useHover,
+  useAnchor,
+  RectAnchor,
+} from '@console/topology';
+import { useAccessReview } from '@console/internal/components/utils';
+import { modelFor, referenceFor } from '@console/internal/module/k8s';
+import SvgBoxedText from '@console/dev-console/src/components/svg/SvgBoxedText';
+import {
+  useDisplayFilters,
+  getTopologyResourceObject,
+  useSearchFilter,
+  NODE_SHADOW_FILTER_ID,
+  NODE_SHADOW_FILTER_ID_HOVER,
+  NodeShadows,
+} from '@console/dev-console/src/components/topology';
+import {
+  VM_STATUS_ERROR,
+  VM_STATUS_IMPORT_ERROR,
+  VM_STATUS_IMPORT_PENDING,
+  VM_STATUS_IMPORTING,
+  VM_STATUS_MIGRATING,
+  VM_STATUS_OFF,
+  VM_STATUS_PAUSED,
+  VM_STATUS_POD_ERROR,
+  VM_STATUS_RUNNING,
+  VM_STATUS_STARTING,
+  VM_STATUS_STOPPING,
+  VM_STATUS_V2V_CONVERSION_ERROR,
+  VM_STATUS_V2V_CONVERSION_IN_PROGRESS,
+  VM_STATUS_V2V_CONVERSION_PENDING,
+  VM_STATUS_VMI_WAITING,
+} from '../../../statuses/vm/constants';
```

Comment:
very recently 2 new more statuses were added: VM_STATUS_V2V_VM_IMPORT_IN_PROGRESS (should go under kubevirt-m-importing) and VM_STATUS_V2V_VM_IMPORT_ERROR (should go under kubevirt-m-error)